tests/beta/test_default_tensor_tf.py

```diff
@@ -602,6 +602,9 @@ def test_autograd_interface(self, circuit, interface, tol):
         res = np.asarray(grad_fn(*self.p))
         assert np.allclose(res, self.expected_grad, atol=tol, rtol=0)
 
+    @pytest.mark.xfail(
+        reason="using a TF device with Torch interface no longer works with the new QNode"
```

Comment:
So odd, why should this ever have worked?

Reply:
Ah, because previously if you used Torch with a TF device, the following would happen:

1. The interface receives Torch tensors, and unwraps them to NumPy arrays
2. The device is evaluated and/or parameter-shift is applied using NumPy arrays
3. The device returns a TF result, which is then converted back to a Torch tensor.

However, with the new interfaces, there is an issue with step (3). The device is returning a _TF tensor_, and the Torch interface is attempting to convert it to a _Torch tensor_.

This works fine in the old interfaces, but is blocked in the new one.

I wasn't sure if this is a common or expected use-case at _all_, so rather than 'fix' it, I think it makes sense just to leave it as unsupported?

I left the tests behind though just in case we want to revisit.

Reply:
> So odd, why should this ever have worked?

Yeah I was wondering the same thing... shouldn't everything eventually (already?)  just be done using `qml.math` so that there is no conversion to/from numpy needed?